### PR TITLE
Use TaskScope

### PR DESCRIPTION
**Pull request by dagguh**
When bypassing the `TaskScope`, we ended up as having insufficient metadata, which ended up as "unknown" tasks in GUI:
image
image

### CODE REVIEW

File: src/test/kotlin/com/atlassian/performance/tools/awsinfrastructure/LogConfiguration.kt [hunk 1]
```diff
@@ -30,11 +30,6 @@ internal class LogConfiguration(
                     null
                 )
             },
-            logToFile(
```
Comment (mgrzaslewicz) [inline src/test/kotlin/com/atlassian/performance/tools/awsinfrastructure/LogConfiguration.kt:33]: Why not have both (events and logs)?

Comment (dagguh) [reply src/test/kotlin/com/atlassian/performance/tools/awsinfrastructure/LogConfiguration.kt:33]: Unnecessary tax and dependency. If a consumer wants logs, they can subscribe and log. They don't need it by default thanks to GUI.

Comment (dagguh) [reply src/test/kotlin/com/atlassian/performance/tools/awsinfrastructure/LogConfiguration.kt:33]: Also, that behavior change is caused by switching from `TaskTimer` to `TaskScope`, not from this log config. This is just removing dead code here.

**Review by mgrzaslewicz — APPROVED** [review]

**Comment by dagguh** [comment]
After:
image

**Pull request merged**